.github/workflows/ci.yml

```diff
@@ -18,18 +18,21 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v2
         with:
-          python-version: 2.7
+          python-version: 3.8
 
       - name: Install Python dependencies
         run: |
           pip install --upgrade pip
-          pip install yadage
+          pip install --pre reana-client
 
-      - name: Validate workflow
-        run: yadage-validate workflow/yadage/workflow.yaml | grep -q 'workflow validates'
```

Comment:
I guess this part is covered by the [`yadage_load` function](https://github.com/reanahub/reana-commons/blob/294c0c59448e8d12c8772a29a5854838b08d4884/reana_commons/yadage.py#L16)?

Just trying to make sure if we can get rid of this safely.

Reply:
Yes I understand I covered by that function